sources/packages/web/src/components/common/SearchStudents.vue

```diff
@@ -0,0 +1,174 @@
+<template>
+  <v-form ref="searchStudentsForm">
+    <content-group class="mb-8">
+      <v-row
+        ><v-col>
+          <v-text-field
+            label="Application number"
+            density="compact"
+            data-cy="appNumber"
+            variant="outlined"
+            v-model="appNumber"
+            @keyup.enter="searchStudents"
+            hide-details
+          />
+        </v-col>
+        <v-col>
+          <v-text-field
+            label="SIN"
+            density="compact"
+            data-cy="sin"
+            variant="outlined"
+            v-model="sin"
+            @keyup.enter="searchStudents"
+            hide-details
+          />
+        </v-col>
+        <v-col>
+          <v-text-field
+            label="Given names"
+            density="compact"
+            data-cy="firstName"
+            variant="outlined"
+            v-model="firstName"
+            @keyup.enter="searchStudents"
+            hide-details
+          /> </v-col
+        ><v-col>
+          <v-text-field
+            label="Last name"
+            density="compact"
+            data-cy="lastName"
+            variant="outlined"
+            v-model="lastName"
+            @keyup.enter="searchStudents"
+            hide-details
+          /> </v-col
+        ><v-col
+          ><v-btn
+            color="primary"
+            class="p-button-raised"
+            data-cy="searchStudents"
+            @click="searchStudents()"
+          >
+            Search
+          </v-btn></v-col
+        >
+      </v-row>
+      <v-input
+        :rules="[isValidSearch()]"
+        hide-details="auto"
+        error
+      /> </content-group
+  ></v-form>
+  <template v-if="studentsFound">
+    <body-header title="Results" />
+    <content-group>
+      <toggle-content :toggled="!students?.length">
+        <DataTable :value="students">
+          <Column field="firstName" header="First Name" :sortable="true">
+            <template #body="slotProps">
+              <div class="p-text-capitalize">
+                {{ slotProps.data.firstName }}
+              </div>
+            </template>
+          </Column>
+          <Column field="lastName" header="Last Name" :sortable="true">
+            <template #body="slotProps">
+              <div class="p-text-capitalize">
+                {{ slotProps.data.lastName }}
+              </div>
+            </template>
+          </Column>
+          <Column field="birthDate" header="Date of Birth">
+            <template #body="slotProps">
+              <div class="p-text-capitalize">
+                {{ dateOnlyLongString(slotProps.data.birthDate) }}
+              </div>
+            </template>
+          </Column>
+          <Column header="Action">
+            <template #body="slotProps">
+              <v-btn
+                color="primary"
+                class="p-button-raised"
+                data-cy="viewStudent"
+                @click="$emit('goToStudentView', slotProps.data.id)"
```

Comment:
As an alternate approach because the params are same between both routes, the component can receive a a route prop and do the navigation here. 
The benefit it is avoid the emit. `<search-students :view-student-route="InstitutionRoutesConst.STUDENT_PROFILE">`

Just a suggestion.

Reply:
Yes, there are many ways of doing that. I'm following https://github.com/bcgov/SIMS/pull/1857#pullrequestreview-1373523716